.github/workflows/ci.yml

```diff
@@ -43,6 +42,19 @@ jobs:
         id: context
         uses: ./.github/actions/context
 
+  actionlint:
```

Comment:
Can we explicitly drop permissions for this ? It's probably a good idea to do that for every job, but especially here since you're using an external docker image...

Reply:
I would do that in a separate issue. Could you file it, not a bad idea to default no permissions.

Reply:
Added that to https://github.com/mozilla/addons/issues/15312 and raised its priority